TINY-10385: fix schema.isWrapper function (#9197)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

### Fixed
- Translation syntax for announcement text in the table grid was incorrectly formatted. #TINY-10141

> - The functions `schema.isWrapper` and `schema.isInline` didn't exclude element names that started with `#` those should not be considered elements. #TINY-10385

Reviewer: ```suggestion
- The functions `schema.isWrapper` and `schema.isInline` did not exclude node names that started with `#` which should not be considered as elements. #TINY-10385
```

Author: changed
